pkg/component/extensions/operatingsystemconfig/original/components/kubelet/component.go

```diff
@@ -103,6 +111,12 @@ WantedBy=multi-user.target
 [Service]
 Restart=always
 RestartSec=5
+` +
+// The default for HTTP2_READ_IDLE_TIMEOUT_SECONDS is 30 and for HTTP2_PING_TIMEOUT_SECONDS 15.
+// This results in issues if the tcp connection to kube-apiserver is silently dropped,
+// as node-monitor-grace-period is only 40s.
+// HTTP2_READ_IDLE_TIMEOUT_SECONDS + HTTP2_PING_TIMEOUT_SECONDS should be less than node-monitor-grace-period.
```

Comment:
Since this period is configurable in the `Shoot` ([ref](https://github.com/gardener/gardener/blob/3ee1b8ad133df44f28dfd3266e1438ec01da577c/example/90-shoot.yaml#L219)), don't we need to compute these timeouts according to what is specified there? `40s` is the Kubernetes default, but people could specify less or more?